keps/sig-cloud-provider/2025-extend-konnectivity-for-both-directions/README.md

```diff
@@ -0,0 +1,694 @@
+<!--
+**Note:** When your KEP is complete, all of these comment blocks should be removed.
```

Comment:
Seems like the metadata has been moved to the kep.yaml, would be nice to get rid of these big comment blocks.